docs/source/reference/command_line.rst

```diff
@@ -105,6 +105,7 @@ Below is a list with all available subcommands.
       disable    Disable the computer for the given user.
       duplicate  Duplicate a computer allowing to change some parameters.
       enable     Enable the computer for the given user.
+      export     Exports the Authinfo details for a computer (and user).
```

Comment:
Why add the `user` here? Only `computer` should be enough, no? I suppose this is because `Computer.get_configuration` allows specifying a user? Though, as `verdi computer export` in its current state doesn't provide that option, we could remove its mentioning here, or, alternatively, allow specifying an optional `user` via `@options.USER`.

Reply:
I wonder if we should phrase even slighly more differently still. What actually happens is that `verdi computer setup` creates a `Computer` instance and `verdi computer configure` then creates a `AuthInfo`, which joins a `Computer` and `User` together. This is because a computer can be configured for multiple users.

So really what we are exporting is a file describing the `Computer` instance and a file describing the `AuthInfo`. Now I think that most users are not really familiar with the `AuthInfo`, so I wonder if we should include it here in the docstring. How about
```
export    Export a computer and its transport configuration
```
Note that exactly what data goes into the `AuthInfo` depends on the chosen transport type for the computer. That is also why `verdi computer configure` has the transport plugin as a subcommand, because that determines which options should be exposed.

I think users are more familiar with transport than `AuthInfo`.

Reply:
For me, "authentication info" seems nicer than `AuthInfo` especially for users who are not yet familiar with the AiiDA classes. If that captures the essence of what is being configured, then that's fine for me.